packages/nodes-base/nodes/Mattermost/v1/methods/loadOptions.ts

```diff
@@ -0,0 +1,150 @@
+import {
+	IDataObject,
+	ILoadOptionsFunctions,
+	INodePropertyOptions,
+	NodeOperationError,
+} from 'n8n-workflow';
+
+import {
+	apiRequest,
+} from '../transport';
+
+const loadOptions = {
```

Comment:
We can do away with the wrapping object and its methods, have just functions at top level, and from `index.ts` we `export * from './loadOptions.ts'`. It would be simpler and in keeping with other files that only contain standalone functions.